secure_message/resources/messages.py

```diff
@@ -89,13 +89,13 @@ def _message_save(message, is_draft, draft_id):
         save = Saver()
         save.save_message(message.data)
         save.save_msg_event(message.data.msg_id, 'Sent')
+        save.save_msg_actors(message.data.msg_id, message.data.msg_from, message.data.msg_to[0], g.user.is_internal)
```

Comment:
Does there need to be any validation to make sure none of these are empty or does it not matter?  I **think** it doesn't matter but I thought I'd ask the question

Reply:
it does matter but validation takes place on entry before getting here

Reply:
When you say takes place on entry, do you mean in the UI?

Reply:
No in the API . In Domain.py. In MessageSchema and DraftSchema ( different rules apply) all messages and drafts are validated close to the point of entry before passing closer to the DB